WooCommerce/Classes/POS/Presentation/Card Present Payments/Reader Messages/PointOfSaleCardPresentPaymentSuccessMessageViewModel.swift

```diff
@@ -1,5 +1,16 @@
 import Foundation
 
 struct PointOfSaleCardPresentPaymentSuccessMessageViewModel {
+    let imageName = "pos_payment_successful"
```

Comment:
nit: could we move the image definition to `UIImage+Woo.swift` so that we can also ensure it's non-nil in the unit tests in `IconTests`?

Also, it would be nice to follow the naming convention in the current images in Images.xcassets using dashes instead of underscores.
```suggestion
    // Moves this to UIImage+Woo to return `UIImage(named: "pos-payment-successful")` and uses the UIImage reference here
    let imageName = "pos-payment-successful"
```

Reply:
I was thinking of just having the name so I can use it like `Image(imageName)` and avoid having UIKit code in the `SwiftUI` view. Using the `UIImage+Woo` property would help to avoid typos with the strings.
Maybe I just have it like this:
```
static var posPaymentSuccessfulImageName: String {
        return "pos-payment-successful"
    }

    static var posPaymentSuccessfulImage: UIImage {
        UIImage(named: posPaymentSuccessfulImageName)!
    }
```

And that way the string is used just in one place and we can reference the name or the `UIImage as we need it.

Reply:
Hm, even with that I would have it `let imageName = UIImage.posPaymentSuccessfulImageName` and would basically have to import UIKit in the view model files.

I think I will add image names to a String extension and have it like that to avoid extra UIKit imports